apps/web/src/serlo-editor/plugins/fill-in-the-blanks-exercise/renderer.tsx

```diff
@@ -78,121 +61,134 @@ export function FillInTheBlanksRenderer(props: FillInTheBlanksRendererProps) {
   /** Maps blankId to the text that should be displayed in the blank.  */
   const textInBlanks = useMemo(() => {
     const newMap = new Map<BlankId, { text: string }>()
-    blanks.forEach((blankState) =>
+    blanks.forEach((blankState) => {
+      const firstCorrectAnswer = blankState.correctAnswers[0]?.answer ?? ''
```

Comment:
```suggestion
      const firstCorrectAnswer = blankState.correctAnswers.at(0)?.answer ?? ''
```

`.at()` already has the type `undefined | A` for a list `A[]`

Reply:
Nice! Did not know about that yet.